.emacs.d/lisp/visual/frame-c.el

```diff
@@ -1,6 +1,7 @@
 ;; disable tool bar and menu bar
 (tool-bar-mode -1)
 (menu-bar-mode -1)
-(scroll-bar-mode -1)
+(if (boundp 'scroll-bar-mode)
```

Comment:
yes according to our style guides, but it looks like there's not a difference. http://www.gnu.org/software/emacs/manual/html_node/elisp/Function-Cells.html